tasks/broad/RNAWithUMIsTasks.wdl

```diff
@@ -101,6 +101,39 @@ task ExtractUMIs {
   }
 }
 
+# Adapter clipping
+task Fastp {
+  input {
+    File fastq1
+    File fastq2
+    String output_prefix
+    File adapter_fasta = "gs://broad-dsde-methods-takuto/RNA/resources/Illumina_adapters.fasta" # sato: move to a public place
+  }
+
+  Int disk_size = 5*ceil(size(fastq1, "GiB")) + 128
+
+  command {
+    fastp --in1 ~{fastq1} --in2 ~{fastq2} --out1 ~{output_prefix}_read1.fastq.gz --out2 ~{output_prefix}_read2.fastq.gz \
+    --disable_quality_filtering --adapter_fasta ~{adapter_fasta} 
+  }
+  
+
+  runtime {
+    docker: "biocontainers/fastp:v0.20.1_cv1"
```

Comment:
what is this tool and is it regularly maintained/updated? If this docker image is something that is an official release we can leave it as is, otherwise we will want to move it into a warp managed image

Reply:
https://github.com/OpenGene/fastp is the tool and the docker comes form biocontainer...definitely not an official release, it was the best I could find.

Reply:
Hmm. I tested with 0.20.1 so we should stick with that. It should be safe to use 0.23.1 since all the updates since 0.20.1 are not related to adapter trimming, but just to be safe.

Reply:
okay sounds good!